### PR TITLE
docs(35/migrations) add portal templates breaking changes

### PR DESCRIPTION
**Pull request by Jlawlzz**
This PR adds steps necessary portal template changes to migrate from 0.34x to 0.35.

### CODE REVIEW

File: app/enterprise/0.35-x/deployment/migrations.md [hunk 1]
```diff
@@ -27,6 +27,17 @@ title: Migrating to 0.35
 '/portal/developers/*/password' => '/developers/*/password'
 '/portal/invite'                => '/developers/invite'
 ```
+* As a result of the switch to serverside rendering, a few portal template files need to be updated or replaced to regain full functionality:
+    1. Replace contents of partial `spec/index-vue` with:
+    https://raw.githubusercontent.com/Kong/kong-portal-templates/master/themes/default-ie11/partials/spec/index-vue.hbs
```
Comment (levfishbluefish) [inline app/enterprise/0.35-x/deployment/migrations.md]: do you want these to be links to the file? or are you intending for people to copy/paste this into a command?

Comment (levfishbluefish) [reply app/enterprise/0.35-x/deployment/migrations.md]: @Jlawlzz

Comment (Jlawlzz) [reply app/enterprise/0.35-x/deployment/migrations.md]: @levfishbluefish These should be links to the file

File: app/enterprise/0.35-x/deployment/migrations.md [hunk 2]
```diff
@@ -27,6 +27,17 @@ title: Migrating to 0.35
 '/portal/developers/*/password' => '/developers/*/password'
 '/portal/invite'                => '/developers/invite'
 ```
+* As a result of the switch to serverside rendering, a few portal template files need to be updated or replaced to regain full functionality:
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/deployment/migrations.md]:
```suggestion
* As a result of the switch to server-side rendering, a few portal template files need to be updated or replaced to regain full functionality:
```

File: app/enterprise/0.35-x/deployment/migrations.md [hunk 3]
```diff
@@ -27,6 +27,17 @@ title: Migrating to 0.35
...
+    1. Replace contents of partial `spec/index-vue` with contents of:
+    [](https://raw.githubusercontent.com/Kong/kong-portal-templates/master/themes/default-ie11/partials/spec/index-vue.hbs)
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/deployment/migrations.md]:
```suggestion
    [https://raw.githubusercontent.com/Kong/kong-portal-templates/master/themes/default-ie11/partials/spec/index-vue.hbs](https://raw.githubusercontent.com/Kong/kong-portal-templates/master/themes/default-ie11/partials/spec/index-vue.hbs)
```

**Pull request merged**